tests/server/routes.js

```diff
@@ -103,6 +103,10 @@ define([
     '/oauth/signin',
     '/oauth/signup',
     '/oauth/force_auth',
+    '/reset_password',
+    '/v1/reset_password',
```

Comment:
reset_password is one of those routes that are allowed to have a `/v1` in front of it.